YPP-0015.md

```diff
@@ -0,0 +1,83 @@
+# Proposal
+Enable UNI as a collateral for DAI and USDC
+
+# Background
+UNI has a $9,697M market cap and 287K holders.
+
+To enable UNI as a collateral we will be using the UNI/ETH chainlink price feed as a source of price data.
+
+# Details
+
+The [addChainlinkCollateral-1.ts](https://github.com/yieldprotocol/environments-v2/blob/c46522044eff41e3d708f11e4f172c4edf80dbae/scripts/governance/addChainlinkCollateral/addChainlinkCollateral-1.ts) and [addChainlinkCollateral-2.ts](https://github.com/yieldprotocol/environments-v2/blob/c46522044eff41e3d708f11e4f172c4edf80dbae/scripts/governance/addChainlinkCollateral/addChainlinkCollateral-2.ts) scripts  will be used, with [this input](https://github.com/yieldprotocol/environments-v2/blob/c46522044eff41e3d708f11e4f172c4edf80dbae/scripts/governance/addChainlinkCollateral/addUNI.config.ts):
```

Comment:
Shouldn't this be scripts specific to Uniswap?

Reply:
Alberto had refactored the code in a way that requires only the config file to be added for a new collateral to be added. Only the config file to be taken changes.

Reply:
The [this input](https://github.com/yieldprotocol/environments-v2/blob/c46522044eff41e3d708f11e4f172c4edf80dbae/scripts/governance/addChainlinkCollateral/addUNI.config.ts) link is specific to uniswap.